### PR TITLE
Added totalAllowedGasPerSec to throttle_definitions.proto

### PR DESCRIPTION
**Pull request by stoqnkpL**
Signed-off-by: Stoyan Panayotov <stoyan.panayotov@limechain.tech>

**Description**:

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by SimiHunjan — APPROVED** [review]

**Review by shemnon — APPROVED** [review]

File: services/throttle_definitions.proto [hunk 1]
```diff
@@ -82,4 +82,6 @@ message ThrottleBucket {
  */
 message ThrottleDefinitions {
   repeated ThrottleBucket throttleBuckets = 1;
+  uint64 totalAllowedGasPerSecFrontend = 2;
+  uint64 totalAllowedGasPerSecConsensus = 3;
```
Comment (shemnon) [inline services/throttle_definitions.proto:86]: This should be snake_case
```suggestion
  uint64 total_allowed_gas_per_sec_frontend = 2;
  uint64 total_allowed_gas_per_sec_consensus = 3;
```

**Comment by shemnon** [comment]
LGTM pending change to snake_case

File: services/throttle_definitions.proto [hunk 2]
```diff
@@ -82,4 +82,6 @@ message ThrottleBucket {
  */
 message ThrottleDefinitions {
   repeated ThrottleBucket throttleBuckets = 1;
+  uint64 totalAllowedGasPerSecFrontend = 2;
```
Comment (tinker-michaelj) [inline services/throttle_definitions.proto:85]: SDK team has requested snake case for all new fields, e.g. 
```
uint64 total_allowed_gas_per_sec_frontend = 2;
```

**Review by tinker-michaelj — CHANGES_REQUESTED** [review]
Just a stylistic change for field naming, otherwise looks good!

File: services/throttle_definitions.proto (hunk 2 above)
Comment (tinker-michaelj) [inline services/throttle_definitions.proto:85]: It's also worth noting that we have an (informal) contract with mirror node operators to **never** remove proto fields, no matter how long they have been deprecated. 😲

So if we have even a 10% doubt that maybe we'll end up following @shemnon's suggestion [here](https://github.com/hashgraph/hedera-services/pull/2373#discussion_r741285719), and using a "normal" throttle bucket for gas throttling, then maybe for release 0.20 we just make these two numbers global/dynamic properties.

But if we're pretty certain about splitting these numbers out in file `0.0.123`, then seems fine.

File: services/throttle_definitions.proto (hunk 1 above)
Comment (stoqnkpL) [reply services/throttle_definitions.proto]: Moved these to global dynamic properties.

File: services/throttle_definitions.proto (hunk 2 above)
Comment (stoqnkpL) [reply services/throttle_definitions.proto]: Moved the gas limit configs to global dynamic properties.

**Review by tinker-michaelj — APPROVED** [review]
LGTM

**Pull request merged**